app/models/concerns/exportable.rb

```diff
@@ -21,16 +21,56 @@ module Exportable
     "Insurance" => :insurance,
     "Income" => :income,
     "Referred By" => :referred_by,
+    "Referred to clinic by fund" => :referred_to_clinic,
     "Appointment Date" => :appointment_date,
     "Initial Call Date" => :initial_call_date,
     "Urgent?" => :urgent_flag,
-    "Special Circumstances" => :special_circumstances 
-  }
+    # "Special Circumstances" => :special_circumstances,
+    "LMP at intake (weeks)" => :last_menstrual_period_weeks,
+    "Race or Ethnicity" => :race_ethnicity,
+    "Employment status" => :employment_status,
+    "Abortion cost" => :procedure_cost,
+    "Patient contribution" => :patient_contribution,
+    "NAF pledge" => :naf_pledge,
+    "Fund pledge" => :dcaf_soft_pledge,
+    "Clinic" => :export_clinic_name,
+    "Pledge sent" => :pledge_sent,
+    "Resolved without fund assistance" => :resolved_without_dcaf,
+    "Pledge generated time" => :pledge_generated_at
+
+    # TODO clinic stuff
+    # TODO call stuff
+    # TODO fulfillment stuff
+    # TODO external pledges
+  }.freeze
 
   def has_alt_contact?
     other_contact.present? || other_phone.present? || other_contact_relationship.present?
   end
 
+  def export_clinic_name
+    clinic.try :name
+  end
+
+  def age_range
+    case age
+    when nil
+      nil
+    when 0..18
+      'Under 18'
+    when 18..24
+      '18-24'
+    when 25..34
+      '25-34'
+    when 35..44
+      '35-44'
+    when 45..54
+      '45-54'
+    when 55..100
+      '55+'
+    end
```

Comment:
The test is a good flag! So is the default case. I made it 'bad value' which seemed okay.